src/PriorityQueue/LegacyImplementation.php

```diff
@@ -27,7 +34,7 @@
  * "inner" iterator in the form of an SplPriorityQueue object for performing
  * the actual iteration.
  */
-class PriorityQueue implements Countable, IteratorAggregate, Serializable
+class LegacyImplementation implements Countable, IteratorAggregate, Serializable
```

Comment:
Should be `@internal` IMO: the implementation should not be part of the public API